randovania/games/samus_returns/logic_database/Area 1.txt

```diff
@@ -1347,12 +1418,6 @@ Extra - asset_id: collision_camera_049
   > Dock to Inner Temple East Hall
       Trivial
 
-> Dock from Inner Temple Teleporter Access; Heals? False
-  * Layers: default
-  * Blocked Passage to Inner Temple Teleporter Access/Dock to Inner Temple Teleporter
-  > Dock to Inner Temple East Hall
-      Trivial
-
```

Comment:
Change the connection from the tunnel to the open passage to just need morph to match the other tunnels in the DB. Also makes the Melee Clip path valid when it's readded.